CONTENT/pitch.markdown

```diff
@@ -5,10 +5,10 @@
 
 ## Come, learn to code with us.
 
-Wizard Amigos Institute invites you to join its co-learning programming sessions in a friendly and playful atmosphere.
-Learn with us how to use state-of-the-art tools and methods to become a self-directed collaborative coder who co-creates the future using special magic programming wizardry.
+Wizard Amigos Institute is a programming school for children. We invite you to join our co-learning programming sessions in a friendly and playful atmosphere.
+Learn with us how to use state-of-the-art tools and methods. Become a self-directed collaborative programmer (=coder) and co-create the future using programming wizardry.
 
```

Comment:
The problem is that I would like to say clearly who we are: Wizard Amigos Institute is a programming school for children. How can I now continue without using WE. I can make this sentence longer and say Wizard Amigos Institute is a programming school for children that invites you... Too long! I will change to sth that I think is ok, check and give feedback.